### DISCUSSION TITLE
Teamsへの通知

### DISCUSSION THREAD

**Discussion by eyelly** [growilabs/growi#6195] [Ideas / Requests]
既にissueの[ユーザーリクエスト](https://github.com/weseek/growi/issues/6164)になっていますが
現行のSlackのように、Teamsへwebhook通知ができればいろんな悩みから解放されます。
(プラグインを自分たちで作れば良いと言われたらぐうの音もでませんが。)

通知周りは社内wiki導入の中で意外と重要どころだと思うのため、こちらで議論させてください。

**Comment by yuki-takei** [comment]
本 discussion と同内容のリクエストが [#7726](https://github.com/growilabs/growi/discussions/7726) として別途起票されており、こちらの方がリアクションが多いため、#7726 に統合いたします。
引き続き [#7726](https://github.com/growilabs/growi/discussions/7726) にてアップボートやコメントをお寄せください！
